installation/security.md

```diff
@@ -1,30 +1,78 @@
 ---
 layout: documentation
-title: Authentication and Encryption through NGINX Reverse Proxy
+title: Securing Communication and Access
 ---
 
 {% include base.html %}
 
-# Running openHAB Behind a Reverse Proxy
+# Securing access to openHAB
+
+openHAB has mainly two ways to be accessed:
+
+1. Through the command line console, which is done through ssh and thus always authenticated and encrypted. You will find all details about this in the [Console documentation](/administration/console).
+2. Through HTTP(S), which we will look at in the following.
+
+{::options toc_levels="2..3"/}
```

Comment:
by intention. Adding level 4 makes the TOC huge (and all the NGINX details are not relevant for most users). I tried to add a 4..5-toc below in the NGINX section, but unfortunately, you can only have one toc per page.